Client/Frontend/Strings.swift

```diff
@@ -1707,6 +1707,45 @@ extension String {
                     tableName: "Settings",
                     value: "Default Search Engine",
                     comment: "Accessibility label for default search engine setting.")
+                public static let LearnAboutSuggestions = MZLocalizedString(
+                    key: "Settings.Search.Accessibility.LearnAboutSuggestions.v122",
+                    tableName: "Settings",
+                    value: "Learn more about %@ Suggest",
+                    comment: "Accessibility label for Learn more about Firefox Suggest. Placeholder is for the app name - Firefox.")
+            }
+
+            public struct Suggest {
+                public static let AddressBarSettingsTitle = MZLocalizedString(
+                    key: "Settings.Search.Suggest.AddressBarSetting.Title.v122",
+                    tableName: "Settings",
+                    value: "Address bar - %@ Suggest",
+                    comment: "In the Search page of the Settings menu, the title for address bar Fx Suggest Settings. Placeholder is for the app name - Firefox.")
+                public static let SuggestionsFromFirefoxTitle = MZLocalizedString(
+                    key: "Settings.Search.Suggest.SuggestionsFromFirefox.Title.v122",
+                    tableName: "Settings",
+                    value: "Suggestions from %@",
+                    comment: "In the Search page of the Settings menu, the title for setting to enable Suggestions from Firefox. Placeholder is for the app name - Firefox.")
+                public static let SuggestionsfromFirefoxDescription = MZLocalizedString(
+                    key: "Settings.Search.Suggest.SuggestionsFromFirefox.Description.v122",
+                    tableName: "Settings",
+                    value: "Get suggestions from the web related to your search",
+                    comment: "In the Search page of the Settings menu, the description for enabling suggestions from Firefox")
```

Comment:
```suggestion
                    comment: "In the Search page of the Settings menu, the description for the setting to enable Suggestions from Firefox")
```

Let's keep the comment consistent between the title and description. The `key` already implies that they're related, but let's describe them the same way in the comment just to make it extra clear.